pkg/oc/cli/admin/release/new.go

```diff
@@ -438,16 +470,37 @@ func (o *NewOptions) Run() error {
 
 		fmt.Fprintf(o.ErrOut, "info: Found %d images in release\n", len(is.Spec.Tags))
 
-	case len(o.FromImageStream) > 0:
+	case len(o.FromImageStream) > 0, len(o.FromImageStreamFile) > 0:
 		is = &imageapi.ImageStream{}
 		is.Annotations = map[string]string{}
 		if len(o.FromImageStream) > 0 && len(o.Namespace) > 0 {
 			is.Annotations[annotationReleaseFromImageStream] = fmt.Sprintf("%s/%s", o.Namespace, o.FromImageStream)
 		}
 
-		inputIS, err := o.ImageClient.ImageV1().ImageStreams(o.Namespace).Get(o.FromImageStream, metav1.GetOptions{})
-		if err != nil {
-			return err
+		var inputIS *imageapi.ImageStream
+		if len(o.FromImageStreamFile) > 0 {
+			data, err := ioutil.ReadFile(o.FromImageStreamFile)
+			if os.IsNotExist(err) {
+				return err
+			}
+			if err != nil {
+				return fmt.Errorf("unable to read input image stream file: %v", err)
+			}
+			is := &imageapi.ImageStream{}
+			if err := yaml.Unmarshal(data, &is); err != nil {
```

Comment:
Why doing this manually when you have resource builder?

Reply:
Release commands don't use resource builder at all.  There is no "magic loading".